ManagedInjector.CLI/ManagedInjector.CLI.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net5.0-windows</TargetFramework>
```

Comment:
I don't think this target framework change is needed, and it limits us to only supporting Windows. I'm currently using a Linux system and may want to support that as well in the future.

Same with the net5.0-windows target later, I don't think it needs to change to net5.0-windows7.0.